app/controllers/partners/dashboards_controller.rb

```diff
@@ -12,9 +12,12 @@ def show
       @upcoming_distributions = ::Partner.find(@partner.diaper_partner_id)
                                          .distributions.order(issued_at: :desc)
                                          .where('issued_at >= ?', Time.zone.today)
+      # 2859 Partner distributions -- limit the prior distributions
```

Comment:
Wondering if we need the comments in here?

Reply:
(Nods) Still trying to find the right balance on the comments.  I can yank the ones you've noted